1_basics.ipynb

```diff
@@ -25,20 +43,57 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# What is HSR?\n",
+    "\n",
+    "HSR (HSR: Human Support Robot) is one of the \"Toyota Partner Robots\" that support human activities, and it will assist independent living at home for people with disabilities.\n",
+    "\n",
+    "In this notebook, you will learn the overview of the robot by using the HSR simulator."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "# Jupyter Notebookの使い方"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# How to use Jupyter Notebook"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "Jupyter NotebookではCellにPythonコマンドを入力し逐次実行していくことができます。\n",
     "\n",
     "例えば以下のコマンドを実行してみましょう。コマンドを実行するにはセルを選択した状態で「Shift+Enter」を押してください。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "In Jupyter Notebook, you can enter Python commands in the Cells and execute them sequentially.\n",
+    "\n",
+    "Let's execute the command below. To execute the command, select the cell and press \"Shift + Enter\":"
```

Comment:
If you are referring to a single command, then the sentence you have is fine.  If here are multiple commands, then the following would be better:

    "Let's execute the commands below. To execute the commands, select the cell and press \"Shift + Enter\":"